sdk/core/core-tracing/lib/utils/global.browser.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+/// <reference lib="dom"/>
```

Comment:
Avoid this if at all possible. /// reference directives have the effect of opting our users in to that complete set of types as if they had specified the lib entry themselves. Sometimes, this is what you want. For us, more often we just want TypeScript not to complain, and don't want to augment our users' types any more than necessary for correctness reasons.

Better to declare stub interfaces up front until we get placeholder types. Could do something like this in a d.ts file:

```typescript
interface Window {};
declare var self: Window & typeof globalThis;
```

Reply:
Can you update the guidance? 

https://azure.github.io/azure-sdk/typescript_design.html#ts-config-lib

> DO NOT use the compilerOptions.lib field. Built in typescript libraries (for example, esnext.asynciterable) should be included via reference directives.

The above also links to the following PR which introduced a /// reference:

https://github.com/Azure/azure-cosmos-js/pull/161/files

Reply:
Lib reference removed in latest commit, @bterlson please take another look